client/src/component/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import "./Button.scss";
+
+import { Props } from "./types";
```

Comment:
Типы лучше импортировать с ключевым словом `type`. Их сразу можно будет отличить от импорта значений + такие импорты будут вырезаться на этапе компиляции

```suggestion
import type { Props } from "./types";
```

Reply:
исправил